src/pages/model.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import App from '../../scripts/flux/components/app'
+
+export default React.createClass({
+  displayName: 'ModelPage',
+  render () {
+    return (
+      <App/>
```

Comment:
Expected an identifier and instead saw '<'.<br>Expected ')' to match '(' from line 7 and instead saw 'App'.<br>Expected an operator and instead saw '/'.<br>Expected an operator and instead saw '>'.<br>Missing semicolon.